exchanges/account/account_test.go

```diff
@@ -300,42 +300,28 @@ func TestBalanceInternalLoad(t *testing.T) {
 	assert.ErrorIs(t, err, common.ErrNilPointer, "should error nil pointer correctly")
 
 	err = bi.load(&Balance{Total: 1, Hold: 2, Free: 3, AvailableWithoutBorrow: 4, Borrowed: 5})
-	assert.ErrorIs(t, err, errUpdatedAtIsZero, "should have not been loaded")
+	assert.ErrorIs(t, err, errUpdatedAtIsZero, "should error correctly when updatedAt is not set")
 
 	now := time.Now()
 	err = bi.load(&Balance{UpdatedAt: now, Total: 1, Hold: 2, Free: 3, AvailableWithoutBorrow: 4, Borrowed: 5})
-	assert.NoError(t, err, "should have been loaded")
+	require.NoError(t, err)
 
 	bi.m.Lock()
```

Comment:
🥃 Aside: You didn't introduce this, but I just realised that this is a single test accessing `bi` directly.
We really don't need to lock it 🤷

Reply:
let’s keep it as is for now.